bot/resources/tags/in-place.md

```diff
@@ -0,0 +1,17 @@
+**Out of Place** and **In Place**
+
+In programming, there are two types of operations: "out of place" operations creates a new object, leaving the original object unchanged. "in place" operations modifies the original object, without creating a new one. These operations return None explicitly.
+
+A common example of these different concepts is seen in the use of the methods `list.sort()` and `sorted(...)`. Using `list.sort()` and attempting to access an element of the list will result in an error. 
+
+```py
+a_list = [3, 1, 2]
+a_new_list = a_list.sort() # This will be None
+print(a_new_list[1]) # This will error because it is NoneType and not a list
+
+a_list = [3, 1, 2]
+sorted(a_list)
+print(a_list[0]) # You may expect 1, but it will print 3
+```
+
```

Comment:
This empty line can be removed, Discord's rendering auto-inserts one after code blocks.